src/main/java/com/connexta/ddf/catalog/direct/Utils.java

```diff
@@ -0,0 +1,52 @@
+package com.connexta.ddf.catalog.direct;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.AttributeType.AttributeFormat;
+import ddf.catalog.data.Metacard;
+import java.nio.charset.Charset;
+import java.util.Base64;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class Utils {
```

Comment:
:pencil2: Could rename this to `MetacardMap` and make the method `convert`.  Or the class to `MetacardTransform` and the method to `toMap` if we expect to have other types of metacard transformations.